portfolio/src/main/webapp/script.js

```diff
@@ -153,8 +156,23 @@ function formatCommentText(comment) {
   return quote;
 }
 
-// Sets value of form submission time to current time in client's timezone
-function inputClientTime(form) {
+/**
+ * Sets value of form submission time to current time in client's timezone,
+ * posts form data to server, reloads comments section and then clears
+ * form in preparation for next entry
+ */
+function submitForm(form) {
   const today = new Date();
-  form.time.value = today.toLocaleString();
+  form.timestamp.value = today.getTime();
+  const formData = JSON.stringify($("#newcommentform").serialize());
+  fetch('/data', {
+    method: 'POST', 
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: formData,
+  }).then(response => {
+      loadComments();
+    }).then(response => 
+    document.getElementById("newcommentform").reset());
```

Comment:
Do you need two different promises?

You can reset the form and load the comments simultaneously, right?